gdcdictionary/schemas/virus_sequence_blastn.yaml

```diff
@@ -4,13 +4,13 @@ id: "virus_sequence_blastn"
 title: Virus Sequence Blastn
 type: object
 namespace: https://chicagoland.pandemicresponsecommons.org
-category: analysis
+category: genomic_data_file
 program: '*'
 project: '*'
 description: >
   Provides result details for blast hit for contigs against the nucleotide blast database using the megablast algorithm.
 additionalProperties: false
-submittable: false
+submittable: true
```

Comment:
what does `submittable` do?

Reply:
I'm not sure how `submittable` is used either. I noticed in GTEx dictionary that all the analysis nodes and clinical nodes (metadata nodes) are submittable false; all the object file nodes are submittable true. Previously all these 6 nodes were analysis nodes, so the submittable was false. Now they become genomic_data_file, I kind of follow the rules to change it to submittable true.

Reply:
okay 🤔 probably worth investigating someday haha